src/mon/NVMeofGwMon.h

```diff
@@ -83,8 +83,11 @@ class NVMeofGwMon: public PaxosService,
   void check_sub(Subscription *sub);
 
 private:
+  bool encode_last_gw_version(MonSession *session, std::string gw_id = "");
   void synchronize_last_beacon();
-
+  static  int is_last_version(uint64_t features) {
+    return HAVE_FEATURE(features, SERVER_SQUID);
+  }
```

Comment:
This and encode_last_gw_version don't seem to be used any longer?

Reply:
correct , need to remove the code